server/express/passport.js

```diff
@@ -19,19 +57,19 @@ const jwtOptions = {
     },
   ]),
   ignoreExpiration: false,
-  secretOrKey: fs.readFileSync('public.pem'),
+  secretOrKey: public_pem,
   algorithms: ['RS256'],
 }
 
 module.exports.preparePassport = () => {
   passport.use(new LoginStrategy({
-    jwtSecret: fs.readFileSync('private.pem'),
```

Comment:
I think using `fs.readFileSync` is actually better. Can you put this back please?